views/article.html

```diff
@@ -53,7 +53,24 @@ <h1 itemprop="headline">{{title}}</h1>
 					{{{htmlBody}}}
 				</div>
 			</article>
-		</main>
+
+			{{#if relatedContent}}
+			<aside class="related-content">
+				<h2 class="related-content__header">Related articles</h2>
+				<ul class="related-content__items">
+					{{#each relatedContent}}
+ 					<li class="related-content__item">
+ 						<div class="related-content__container">
+							{{{date}}}
+							<a class="related-content__link" href="/content/{{{id}}}">{{{title}}}</a>
```

Comment:
These should link to ft.com. Relative links won't work as the page won't be viewed from amp.ft.com